src/main/java/frc/robot/RobotContainer.java

```diff
@@ -751,27 +767,52 @@ private void configureSubsystemCommands() {
             Commands.sequence(
                 intake.runOnce(() -> intake.stopIntake()), claw.runOnce(() -> claw.stopClaw())));
 
+    Command nonAutoPluck =
+        Commands.runOnce(() -> isPlucking = true)
+            .andThen(
+                Commands.deadline(
+                    Commands.sequence(
+                        new WaitCommand(0.25),
+                        intake.runOnce(
+                            () -> {
+                              ArmevatorPose setPose =
+                                  isPluckTargetHighSupplier.getAsBoolean()
+                                      ? ArmevatorPose.ALGAE_L3_PLUCK
+                                      : ArmevatorPose.ALGAE_L2_PLUCK;
+
+                              if (isFullAuto) {
+                                setPose = inferPluckArmevatorPose();
+                              }
+                              intake.setTargetPose(setPose);
+                            }),
+                        armevator.runOnce(
+                            () -> {
+                              ArmevatorPose setPose =
+                                  isPluckTargetHighSupplier.getAsBoolean()
+                                      ? ArmevatorPose.ALGAE_L3_PLUCK
+                                      : ArmevatorPose.ALGAE_L2_PLUCK;
+
+                              if (isFullAuto) {
+                                setPose = inferPluckArmevatorPose();
+                              }
+
+                              armevator.setTargetPose(setPose);
+                            }))),
+                claw.run(() -> claw.ejectCoral()))
+            .andThen(claw.run(() -> claw.intakeAlgae()));
+
+    Command autoPluckCommand =
+        Commands.sequence(
+            new ConditionalCommand(
+                new WaitCommand(0.5), new WaitCommand(0), oi.scoreCoralButton()::getAsBoolean),
```

Comment:
see this comment https://github.com/FRC1732/2025-1732Bot/commit/fa31e2e1653eb2629362be89e9351c75352539ce#r153539271